ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -48,6 +50,15 @@ spec:
           --pyxis-url $(params.pyxis_url) \
           --cert-project-id $(params.cert_project_id)
 
+        CURRENT_CERTIFICATION_STATUS=$(cat cert_project | jq '.certification_status'| tr -d '"')
```

Comment:
You can probably remove the `| tr -d '"'` portion of this command if you use `jq -r`.